arch/arm/soc/nxp_imx/rt/soc.c

```diff
@@ -58,6 +58,10 @@ static ALWAYS_INLINE void clkInit(void)
 	/* Setting the VDD_SOC to 1.5V. It is necessary to config AHB to 600Mhz
 	 */
 	DCDC->REG3 = (DCDC->REG3 & (~DCDC_REG3_TRG_MASK)) | DCDC_REG3_TRG(0x12);
+	/* Waiting for DCDC_STS_DC_OK bit is asserted */
+	while (DCDC_REG0_STS_DC_OK_MASK !=
+			(DCDC_REG0_STS_DC_OK_MASK & DCDC->REG0)) {
```

Comment:
I think we usually put a ';' in the body, just to make it clear

```
while (foo) {
   ;
}
```

Reply:
sure, will do.